.eslintrc.json

```diff
@@ -45,7 +45,7 @@
     "no-spaced-func": 2,
     "no-trailing-spaces": 2,
     "operator-linebreak": 2,
-    "quotes": [2, "single"],
+    "quotes": [2, "single", { "avoidEscape": true }],
```

Comment:
This is because on commit - the eslint was complaining:
<img width="453" alt="Screenshot 2020-08-28 at 11 48 14" src="https://user-images.githubusercontent.com/392513/91553212-ecd99b80-e924-11ea-88e9-f4d9352d6a6d.png">